pkg/kubelet/cri/remote/remote_runtime.go

```diff
@@ -676,7 +676,15 @@ func (r *remoteRuntimeService) execSyncV1alpha2(ctx context.Context, containerID
 		if status.Code(err) == codes.DeadlineExceeded {
 			err = exec.NewTimeoutError(fmt.Errorf("command %q timed out", strings.Join(cmd, " ")), timeout)
 		}
-
+		// #106682 if the command doesn't exist in the container the error is nested into the GRPC error
+		// unwrap the error from the message
+		execMsg := strings.SplitAfterN(err.Error(), "starting container process caused: ", 2)
```

Comment:
The message "starting container process caused" depends on the runtime stack and other OCI runtimes may return a different message here.

Reply:
yeah, I didn't like it either, it is too brittle to match on string convention